### DISCUSSION TITLE
Why i am forced to create a team ?

### DISCUSSION THREAD

**Discussion by jchesne** [vercel/community#4381] [Account Issue]
### Question

Hi,

I dont know why when i log in into dashboard, i'm asked to create a team immediately ?
I'm connecting with my github account which is part of my own organisation on github but i dont try to connect with my organisation, only with my personnal account.

Is it normal ?

**Comment by amyegan** [comment]
Hi @jchesne. Can you share a screenshot showing what you see and where it appears in your dashboard? I want to make sure I understand what is happening 🙏

**Comment by jchesne** [comment]
As soon as i click on Start deploying for Hoby plan on price table, i am redirected to the dashboard asking me to create a team.
<img width="1095" alt="Capture d’écran, le 2023-10-07 à 14 36 29" src="https://github.com/vercel/community/assets/1859737/15543a57-4a3b-4f9f-b528-ba125023b85e">
